.gitignore

```diff
@@ -55,3 +55,6 @@ docs/_build/
 
 # PyBuilder
 target/
+
+# Sqlite databases
+*.db
```

Comment:
I had multiple examples at that time, one without access control and one with it. I have removed it so that it does not depend on the deprecated invenio_records/permissions.py but I might add another example providing its own permission_factories later.